src/actions/importing/aioseo-posts-importing-action.php

```diff
@@ -55,6 +56,20 @@ protected function aioseo_exists() {
 		return $this->wpdb_helper->table_exists( $this->get_table() ) === true;
 	}
 
+	/**
+	 * Returns whether the AISOEO post importing action is enabled.
+	 *
+	 * @return bool True if the AISOEO post importing action is enabled.
+	 */
+	public function is_enabled() {
+		$aioseo_importer_conditional = \YoastSEO()->classes->get( AIOSEO_V4_Importer_Conditional::class );
```

Comment:
Maybe just return the result directly.

```php
return \YoastSEO()->classes->get( AIOSEO_V4_Importer_Conditional::class )->is_met();
```

Reply:
I went with the middle ground here for readability purposes, returning the `return $aioseo_importer_conditional->is_met();` 

Let me know if you agree with that :)

Reply:
👍